{"uuid":"e17acf9e-b144-44d6-811e-b8ec329fa1b7","name":"Scenario: User enters Invalid Python code and Presented with Error Message","children":["1950081d-bea2-4231-8eab-0a9d73c7ebd9"],"befores":[],"afters":[{"name":"hooks.Hook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1681612170703,"stop":1681612170704},{"name":"hooks.Hook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1681612171067,"stop":1681612171068},{"name":"hooks.Hook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1681612171213,"stop":1681612171214},{"name":"hooks.Hook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1681612171305,"stop":1681612171306},{"name":"hooks.Hook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1681612171336,"stop":1681612171337}],"start":1681612170512,"stop":1681612171338}